common/main/java/com/couchbase/lite/AbstractReplicatorConfiguration.java

```diff
@@ -144,7 +140,7 @@ public final ReplicatorConfiguration setChannels(@Nullable List<String> channels
     /**
      * Sets the the conflict resolver.
      *
-     * @param conflictResolver A conflict resolver.
+     * @param conflictResolver The replicator type.
```

Comment:
Wrong description. Can you revert this back to A conflict resolver?

Reply:
Not a regression.  That comment has been there for ages.  WIll fix in 3.0